nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters
# Zm myposts - View Current User's Posts

# Description
Developed two features:

 1)  feature that will allow the user to view only the posts they authored

# Ticket Reference
> [#4 - View Current User's Posts](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/4)


## Type of change


# Testing Instructions
- Run the API in Visual studio, and `npm serve` the client folder.
- Type `localhost:3000/users` into the browser bar.
- Start program with ISS Express
- Go to localhost:5001 in your browser.
-Login with "admin@example.com"
- Navigate to the `Posts` link in the navbar.
- You will see the list of all approved posts
- Now navigate to the `My Posts` link in the navbar.
You should now see only the posts that the current sessionUser authored


# Checklist:
